src/frontend/src/btc/services/btc-pending-sent-transactions.services.ts

```diff
@@ -0,0 +1,34 @@
+import { btcPendingSentTransactionsStore } from '$btc/stores/btc-pending-sent-transactions.store';
+import { getPendingBtcTransactions } from '$lib/api/backend.api';
+import type { NetworkId } from '$lib/types/network';
+import { mapNetworkIdToBitcoinNetwork, mapToSignerBitcoinNetwork } from '$lib/utils/network.utils';
+import type { Identity } from '@dfinity/agent';
+
+export const loadBtcPendingSentTransactions = async ({
```

Comment:
For consistency can you return a `ResultSuccess`?

Reply:
Done!